.rubocop.yml

```diff
@@ -1,260 +1,17 @@
-# This (http://c2.com/cgi/wiki?AbcMetric) is super obnoxious
-AbcSize:
-  Enabled: false
-
-AccessorMethodName:
-  Enabled: false
-
-Alias:
-  Enabled: false
+require:
```

Comment:
I started by copying the latest `.rubocop.yml` from Henchman but realized there was a lot of cruft still in there, including some references to Panorama-specific things. So instead I ended up deleting everything and adding rules back in when they were useful for this codebase. It turns out we don't need that many!